internal/api/v1/environment.go

```diff
@@ -0,0 +1,129 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/flexprice/flexprice/internal/api/dto"
+	"github.com/flexprice/flexprice/internal/logger"
+	"github.com/flexprice/flexprice/internal/service"
+	"github.com/flexprice/flexprice/internal/types"
+	"github.com/gin-gonic/gin"
+)
+
+type EnvironmentHandler struct {
+	service service.EnvironmentService
+	log     *logger.Logger
+}
+
+func NewEnvironmentHandler(service service.EnvironmentService, log *logger.Logger) *EnvironmentHandler {
+	return &EnvironmentHandler{service: service, log: log}
+}
+
+// @Summary Create an environment
+// @Description Create an environment
+// @Tags environments
+// @Accept json
+// @Produce json
+// @Security BearerAuth
+// @Param environment body dto.CreateEnvironmentRequest true "Environment"
+// @Success 201 {object} dto.EnvironmentResponse
+// @Failure 400 {object} errors.ErrorResponse
+// @Failure 500 {object} errors.ErrorResponse
```

Comment:
I think `ErrorResponse`  is already in `api/v1` and not a separate package so we need to use just `ErrorResponse` here

Reply:
Once fixed - do run `make swagger` to generate the API docs as well for these APIs